torch/_inductor/codegen/wrapper.py

```diff
@@ -325,8 +338,9 @@ def __init__(self):
         self.last_seen_device_guard_index = None
         self.supports_intermediate_hooks = True
         self.expr_printer = pexpr
-        self.cached_thread_locals = set()
         self.user_defined_kernel_count = 0
+        self.allow_stack_allocation = None
```

Comment:
Do we initialize ```allow_stack_allocation``` with ```None``` on purpose? If so, wondering what the difference would be between ```None``` and ```False``` for this flag.

Reply:
None is "not decided yet", False is "decided false"